<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaperDlg</class>
 <widget class="QDialog" name="PaperDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>论文信息管理系统</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>论文(&amp;L)：</string>
     </property>
     <property name="buddy">
      <cstring>tableView_2</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView_2"/>
   </item>
   <item row="1" column="1" rowspan="3">
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="12" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>退出(&amp;Q)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>论文名(&amp;N)：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_2</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_12">
        <property name="text">
         <string>返回全表(&amp;E)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>确认添加(&amp;Y)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>添加相关文献(&amp;Z)</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>删除相关文献(&amp;V)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>添加论文信息(&amp;A)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>删除选中论文(&amp;D)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>查找(&amp;S)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>与选中论文（选中论文）相关的文献(&amp;P)：</string>
     </property>
     <property name="buddy">
      <cstring>tableView</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView_2</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_10</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_11</tabstop>
  <tabstop>pushButton_12</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PaperDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
